doc/connection.rst

```diff
@@ -249,3 +249,32 @@ The only methods discussed below are those that will be overwritten by the
 .. method:: _wait_for_boot(self)
 
     Wait for the gem5 simulated system to have booted and finished the booting animation.
+
+
+.. _multiple-connections:
+
+Multipe Connections With One Target
```

Comment:
Is it worth adding a note somewhere that Gem5Connections and TelnetConnections do not support this?

Reply:
Good point. I'll add a note for `Gem5Connection`. `TelnetConnection` actually derives from `SshConnection` so it would be supported.

Reply:
In which case for `TelnetConnection`, a call to the `BaseConnection` `initi` method will also need to be added.

Reply:
Ah, yes. Good catch. Fixed.